metaflow/tracing/span_exporter.py

```diff
@@ -0,0 +1,62 @@
+import sys
+from metaflow.metaflow_config import (
+    OTEL_ENDPOINT,
+    ZIPKIN_ENDPOINT,
+    CONSOLE_TRACE_ENABLED,
+)
+
+from opentelemetry.exporter.otlp.proto.http.trace_exporter import (
+    OTLPSpanExporter,
+)
+from opentelemetry.exporter.zipkin.proto.http import ZipkinExporter
```

Comment:
These imports could possibly be made conditional.  With the current implementation I have to install `opentelemetry.exporter.zipkin` as a dependency even if I am not using Zipkin for telemetry, or my stdout fills with import errors due to missing dependencies. Likely same story for Zipkin users getting errors for `opentelemetry.exporter.otlp`